core/templates/dev/head/base.html

```diff
@@ -350,6 +350,8 @@
       {{ include_js_file('expressions/evaluator.js') }}
       {{ include_js_file('expressions/parser.js') }}
 
+      {{ include_js_file('domain/utilities/UrlInterpolationService.js') }}
```

Comment:
Ack. The thing that needs this file is the 'create exploration' button, though, which shows up on pretty much all pages, so I figured it was justified.

More generally, it's actually quite unclear to me how to handle the case of a JS file being included because a service needs a dependency, since the service is not tied to a particular page. I don't know what the solution is, but I want to flag this issue, since it was one of my considerations when trying to figure out what to do here.